.github/workflows/integration.yml

```diff
@@ -131,16 +146,15 @@ jobs:
                 _run sudo multipass set local.driver=lxd
               fi
 
-              # Give multipassd time to settle
-              _retry 12 5 _run $MP find
+              # Give multipassd time to settle (#1995)
+              _retry_grep 12 5 "\w,lts\W" _run $MP find
```

Comment:
Note this may include a reboot on Windows, and the LXD driver is slow on startup, too (#1599). And on Multipass startup, we download the streams and such… so yeah.

Reply:
OK got it (except reboot on Windows, but I'll ask elsewhere).